inc/namespace.php

```diff
@@ -268,6 +271,86 @@ function noop_wp_query_found_rows_on_failed_ep_request( string $sql, WP_Query $q
 	return '';
 }
 
+/**
+ * Modify the mime type query to support truncated forms.
+ *
+ * @param array $query The Elasticsearch query.
+ * @param array $args The WP_Query query vars.
+ * @return array
+ */
+function fix_mime_type_query( array $query, array $args ) : array {
+	if ( empty( $args['post_mime_type'] ) ) {
```

Comment:
Just a simplification: Join both ifs:
```suggestion
	if ( empty( $args['post_mime_type'] ) || ! isset( $query['post_filter'] ) ) {
```

Reply:
More terse but I find things like this tend to be personal preference. Cheers the review though, will get this backported asap